server/cache_test.go

```diff
@@ -625,7 +626,11 @@ func mustSaveStores(c *C, kv *core.KV, n int) []*metapb.Store {
 func mustSaveRegions(c *C, kv *core.KV, n int) []*metapb.Region {
 	regions := make([]*metapb.Region, 0, n)
 	for i := 0; i < n; i++ {
-		region := &metapb.Region{Id: uint64(i)}
+		region := &metapb.Region{
+			Id:       uint64(i),
+			StartKey: []byte(fmt.Sprintf("%d", i)),
+			EndKey:   []byte(fmt.Sprintf("%d", i+1)),
```

Comment:
Consider i is 9, EndKey will be less than StartKey. Better to use '%20d' instead of '%d'.

Reply:
why not using bigendian format directly?

Reply:
seem you should abstract a function to initialize the region.

Reply:
Binary format is hard to read.